packages/block-library/src/navigation/edit/navigation-menu-selector.js

```diff
@@ -60,7 +64,7 @@ export default function NavigationMenuSelector( {
 	return (
 		<>
 			<ExistingMenusOptions
```

Comment:
Very minor, but I don't think `ExistingMenuOptions` needs to be in a separate file now that it's only used in `NavigationMenuSelector`. It can probably be moved into the `navigation-menu-selector.js` file as a separate component. 

Or alternatively it probably doesn't need to be a component any more. The menu groups it renders could be inlined here.

Reply:
Yeh I guess. I mean, it felt a lot cleaner but now everything is normalized perhaps it's OTT to be having to pass props around for no gain.

Reply:
I've now inline all of this and remove the component.